packages/okta-angular/README.md

```diff
@@ -37,6 +37,7 @@ The `OktaAuthModule` is the initializer for your OpenID Connect client configura
   - `clientId` **(required)**: The OpenID Connect `client_id`
   - `redirectUri` **(required)**: Where the callback is hosted
   - `scope` *(optional)*: Reserved for custom claims to be returned in the tokens
+  - `responseType` *(optional)*: Desired token grant types
```

Comment:
I feel like we should be explicitly mentioning that these are string values.

Reply:
I thought about noting that in this section, but seems weird to only mention the types on this configuration value. When this value is evaluated, typescript will error due to a type difference.

Reply:
Aw, types 💘